core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-field-dragabble-item/content-type-field-dragabble-item.component.ts

```diff
@@ -52,18 +66,66 @@ export class ContentTypesFieldDragabbleItemComponent implements OnInit {
             }
         ]
             .filter((field) => field.value)
-            .map((field) => field.name)
-            .join('&nbsp;&nbsp;&#8226;&nbsp;&nbsp;');
+            .map((field) => field.name);
+
+        this.icon = this.fieldService.getIcon(this.field.clazz);
+
+        this.shouldResize = this.fieldAttributes.length > 1;
+
+        if (this.shouldResize) {
+            this.resizeObserver$ = new ResizeObserver((entries) => {
+                const [host] = entries;
+
+                if (host.contentRect.width < 300 && !this.small) {
+                    this.small = true;
+
+                    this.cd.detectChanges();
+                } else if (host.contentRect.width > 300 && this.small) {
+                    this.small = false;
+
+                    this.cd.detectChanges();
+                }
+            });
+
+            this.resizeObserver$.observe(this.el.nativeElement);
```

Comment:
This is clever! until we can add container queries!

Reply:
I even think about making this a directive, like it receives breakpoints and emits the current breakpoint (small, medium, large)

Breakpoints like: 

```javascript
const breakpoints = {
    small: 100, // Less than 100px is small, emits "small"
    medium: 300
    large: 500
}
```

Something like that